web/models.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web
+
+import (
+	"time"
+
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+)
+
+// Driver config is used for direct database connection
+type DriverConfig struct {
+	Driver   string `json:"Driver"`
+	Host     string `json:"Host"`
+	Port     string `json:"Port"`
+	Database string `json:"Database"`
+	User     string `json:"User"`
+	Password string `json:"Password"`
+}
+
+// Dump config is used for dump file approach
```

Comment:
Missing full stop (here and in other places in the file).
Comments should be complete sentences, including punctuation.

Reply:
added full stop and at other places as well